src/main/java/com/ase/restservice/model/Stock.java

```diff
@@ -55,7 +57,15 @@ public void setStockId(final String stockId) {
    */
   @Column(name = "price", nullable = false)
   public Float getPrice() {
-    return price;
+    try {
+        yahoofinance.Stock apiStock = YahooFinance.get(this.stockId);
```

Comment:
Can we move all YahooFinance functionality to a separate service named FinanceService.java I think this Stock model should only represent the stock table we have in the database. So, when getPrice() is called, it should just reply with the price in the database. If we have a separate service and endpoints for the Finance API, administrators of our server can visit /finance/updateStockPrices, and then the FinanceService would retrieve all the stocks from stockService.listStocks() and update each one's price through stockService.updateStockPrice()